.env.sample

```
@@ -8,6 +8,9 @@ NEXT_PUBLIC_STACK_API_KEY=
 NODE_ENV=development
 PINATA_JWT=
 NEXT_PUBLIC_GATEWAY_URL=
+# Lighthouse IPFS (primary storage) - https://lighthouse.storage/
+NEXT_PUBLIC_LIGHTHOUSE_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_LIGHTHOUSE_API_KEY key should go before the NEXT_PUBLIC_REOWN_PROJECT_ID key

(UnorderedKey)